R/ipr2viz.R

```diff
@@ -152,35 +152,35 @@ ipr2viz <- function(infile_ipr = NULL, infile_full = NULL, accessions = c(),
     # Filter by Top Accessions per Accession per DomArch and Lineage
     ipr_out <- subset(
         ipr_out,
-        ipr_out$AccNum %in% top_acc
+        ipr_out$.data$AccNum %in% top_acc
```

Comment:
This use of .data is unnecessary. Here, we can already tell that `AccNum` comes from `ipr_out` as it was explicitly defined as `ipr_out$AccNum`